package.json

```diff
@@ -70,8 +70,7 @@
     "eslint-plugin-standard": "^2.0.0",
     "eslint-plugin-underscore": "0.0.10",
     "istanbul": "1.1.0-alpha.1",
-    "phantomjs": "1.9.x",
-    "phantomjs-polyfill": "0.0.x",
+    "phantomjs": "^2.1.7",
```

Comment:
This package is deprecated, we should move to `phantomjs-prebuilt`, which has a newer version.  This will also require changing the executable path in the test runner.